.travis.yml

```diff
@@ -21,14 +17,13 @@ before_install:
 install: >
   mvn de.qaware.maven:go-offline-maven-plugin:1.2.7:resolve-dependencies
   -DfailOnErrors -DdownloadSources=false
-  -Prun-code-coverage,sonarcloud-analysis
+  -Prun-code-coverage
```

Comment:
The `run-code-coverage` here and on the line 26 is now good for nothing. Shall we keep the .travis.yml or delete it altogether?

Reply:
Good catch, thanks! I'll remove the profile.

Yes, I think I'm going to abandon Travis CI completely. Step by step.